diary-alexander-jaffray/modernized/15-notes-diary.adoc

```diff
@@ -1302,10 +1342,13 @@ a place of honour more than of business.
 
 By favour of Richard Jaffray of London,
 the present hereditary descendant of the Jaffray family,
-the following copy of the original instrument is subjoined, conveying,
-under the great seal of the Commonwealth,
+the following copy of the _original instrument_ is subjoined, conveying,
+under _the great seal of the Commonwealth,_
 the above-mentioned office of Director of the Chancellary to Alexander Jaffray.
 
+[.embedded-content-document]
+--
+
```

Comment:
It won't let me put the note at the right spot, but down a couple paragraphs, where it starts with “Therefore we, with advice and consent of the Commissioners of our said exchequer,... Lets kill the quotation mark before Therefore. 👍🏻